pkg/trace/stats/client_stats_aggregator.go

```diff
@@ -116,6 +116,7 @@ func (a *ClientStatsAggregator) getAggregationBucketTime(now, bs time.Time) (tim
 }
 
 func (a *ClientStatsAggregator) add(now time.Time, p pb.ClientStatsPayload) {
+	resolveContainerTags(&p)
```

Comment:
the resolution is done at the flush step of the concentrator, but at the add step of the clientStatsAggregator. Isn't it a bit confusing?
We could potentially do it in the writer also. Not sure which one is better.

Reply:
Done!